copy/multiple.go

```diff
@@ -230,6 +230,16 @@ func (c *copier) copyMultipleImages(ctx context.Context) (copiedManifest []byte,
 			logrus.Debugf("Copying instance %s (%d/%d)", instance.sourceDigest, i+1, len(instanceCopyList))
 			c.Printf("Copying image %s (%d/%d)\n", instance.sourceDigest, i+1, len(instanceCopyList))
 			unparsedInstance := image.UnparsedInstance(c.rawSource, &instanceCopyList[i].sourceDigest)
+			singleImageOpts := copySingleImageOptions{requireCompressionFormatMatch: false}
```

Comment:
This value is not _actually_ used AFAICS

I would also prefer it to be set in `prepare…`; that way we can unit-test this. (I’m _not_ saying that a unit test is required.)

Reply:
done, added test.